hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/JetSqlSerializerHook.java

```diff
@@ -106,15 +106,12 @@
 
 import java.util.Map;
 
-import static com.hazelcast.internal.serialization.impl.FactoryIdHelper.JET_SQL_DS_FACTORY;
-import static com.hazelcast.internal.serialization.impl.FactoryIdHelper.JET_SQL_DS_FACTORY_ID;
-
 /**
  * Serialization hook for Jet SQL engine classes.
  */
 public class JetSqlSerializerHook implements DataSerializerHook {
 
-    public static final int F_ID = FactoryIdHelper.getFactoryId(JET_SQL_DS_FACTORY, JET_SQL_DS_FACTORY_ID);
```

Comment:
Any change here I'm fine with from a performance perspective since this is done statically (at the moment the class is loaded).